rst/cartridge_dev.rst

```diff
@@ -1215,22 +1260,53 @@ Start/stop using ``systemctl``
 
 .. _cartridge-run-systemctl-config:
 
-When running instances with ``systemctl``, you can specify instance configuration
-in a YAML file.
+When running instances with ``systemctl``, keep these practices in mind:
+
+* You can specify *instance configuration* in a YAML file.
+
+  This file can contain `these options <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge.argparse/#tables>`_;
+  see an example `here <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#usage-example>`_).
+
+  Save this file to ``/etc/tarantool/conf.d/`` (the default ``systemd`` path)
+  or to a location set in the ``TARANTOOL_CFG`` environment variable
+  (if you've edited the application's ``systemd`` unit file).
+  The file name doesn't matter: it can be ``instances.yml`` or anything else you like.
+
+  Here's what ``systemd`` is doing further:
+
+  * obtains ``app_name`` (and ``instance_name``, if specified)
+    from the name of the application's ``systemd`` unit file
+    (e.g. ``APP_NAME@default`` or ``APP_NAME@INSTANCE_1``);
+  * sets default console socket (e.g. ``/var/run/tarantool/APP_NAME@INSTANCE_1.control``),
+    PID file (e.g. ``/var/run/tarantool/APP_NAME@INSTANCE_1.pid``)
+    and ``workdir`` (e.g. ``/var/lib/tarantool/<APP_NAME>.<INSTANCE_NAME>``).
+    Environment=TARANTOOL_WORKDIR=${workdir}.%i
+    Q: what's the workdir for 2 cases:
+    no instance name specified and multi-instance setup?
```

Comment:
Don't overcomplicate it. Curious users can check themselves.

Reply:
Okay. Removing the question then.